DESCRIPTION

```diff
@@ -20,31 +22,31 @@ Imports:
     dplyr,
     DT,
     glue,
-    gsm,
+    gsm (>= 2.1.0),
     htmltools,
     htmlwidgets,
+    jsonlite,
+    kableExtra,
     magrittr,
     purrr,
-    qs,
     rlang,
-    shiny,
+    shiny (>= 1.6.0),
+    shinycssloaders,
     shinyjs,
-    stringr,
-    tidyr
+    yaml
 Suggests: 
-    clindata,
+    chromote (>= 0.3.1),
+    devtools,
     here,
-    pkgload,
     shinytest2,
     testthat (>= 3.0.0),
-    usethis
+    usethis,
+    withr
 Remotes:
-    clindata=Gilead-BioStats/clindata,
-    gsm=Gilead-BioStats/gsm@v1.9.2,
-    qs=traversc/qs
+    gsm=Gilead-BioStats/gsm@main
```

Comment:
do you want it to be pegged to @main or @v2.1.0?

Reply:
Good call! I think we're safest if we peg to a specific release. I'll update in dev.